cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -25,9 +27,16 @@ public Publication generatePublication() {
                    .withAdditionalIdentifiers(Set.of(extractIdentifier()))
                    .withEntityDescription(generateEntityDescription())
                    .withCreatedDate(extractEntryCreationDate())
+                   .withPublisher(extractOrganization())
+                   .withOwner(SOME_REAL_OWNER)
                    .build();
     }
 
+    private Organization extractOrganization() {
```

Comment:
temporary value for registering resources. Objective here is to write things to dynamo without necessarily the correct data